cypress/integration/Tabs.spec.ts

```diff
@@ -4,13 +4,14 @@ describe('Tabs', () => {
   before(() => {
     h.stories.visit();
   });
-  ['Simple', 'NamedKeys'].forEach(story => {
+
+  ['Simple', 'Named Keys'].forEach(story => {
     context(`given the '${story}' story is rendered`, () => {
       beforeEach(() => {
         h.stories.load('Testing/React/Labs/Tabs', story);
       });
 
-      it('should pass accessibility checks', () => {
+      it('should pass axe checks', () => {
```

Comment:
We should probably change all these to specifically call out "axe" vs "accessibility". We're checking "axe" here. The rest of the file checks accessibility.